web/src/utils/constants.ts

```diff
@@ -60,7 +60,6 @@ interface EnvModelConfig {
  */
 export const MODEL_CONFIG: EnvModelConfig = (() => {
   const modelName = process.env.REACT_APP_MODEL;
-  log.fatal(modelName);
```

Comment:
This was a caught in a previous PR but it was already handled in this branch. This log can be taken out, was for debugging purposes.